SPECS/gflags/gflags.spec

```diff
@@ -0,0 +1,64 @@
+Name:           gflags
+Summary:        The gflags package contains a C++ library that implements commandline flags processing. 
+Version:        2.2.2
+Release:        3%{?dist}
+License:        BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://gflags.github.io/gflags/
+Source0:        https://github.com/gflags/gflags/archive/v%{version}.tar.gz
```

Comment:
Nit: Instances of `gflags` in the Source0 line could be replaced with `%{name}`, to be like our other specs.